### PR TITLE
Google Provider Added

### PR DESCRIPTION
**Pull request by fatihkahveci**
for #7

Google Provider added also i change duplicate connectAction method.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,7 +123,7 @@ class FacebookController extends Controller
      *
      * @Route("/connect/facebook/check", name="connect_facebook_check")
      */
-    public function connectAction(Request $request)
+    public function connectCheckAction(Request $request)
```
Comment (weaverryan) [inline README.md]: Ah, good catch - thank you :)

File: DependencyInjection/Providers/GoogleProviderConfigurator.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+namespace KnpU\OAuth2ClientBundle\DependencyInjection\Providers;
+
+use Symfony\Component\Config\Definition\Builder\NodeBuilder;
+
+class GoogleProviderConfigurator implements ProviderConfiguratorInterface
+{
+    public function buildConfiguration(NodeBuilder $node)
+    {
+        // no custom options
```
Comment (weaverryan) [inline DependencyInjection/Providers/GoogleProviderConfigurator.php]: What about `hostedDomain` - I see that here: https://github.com/thephpleague/oauth2-google. It looks like there's also an `accessType`: https://github.com/thephpleague/oauth2-google/blob/master/src/Provider/Google.php#L20. I'm not familiar with either of these - I just spotted them on the driver :)

Comment (fatihkahveci) [inline DependencyInjection/Providers/GoogleProviderConfigurator.php]: @weaverryan Right hostedDomain and accsesType just optional parameters. But we must add this your right my bad :+1:

**Comment by weaverryan** [comment]
Merged! Thanks so much! Btw, if you want to add others (which would be awesome), you don't need to update the README - it updates itself via my super awesome (super messy) script: https://github.com/knpuniversity/oauth2-client-bundle/blob/master/bin/update_readme

Thanks!

**Comment by fatihkahveci** [comment]
@weaverryan This reminde me https://github.com/NARKOZ/hacker-scripts that script :D.

**Pull request merged**